feature-odyssey/src/main/res/drawable/ic_claim_covered.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="17dp"
+    android:height="22dp"
```

Comment:
This isn't 24.dp like those usually are, is this from figma? Looks a bit odd for it to be this way no?